data/styles/clear/include/Basemap_label.mapcss

```diff
@@ -329,14 +329,17 @@ node|z13-[place=city][capital!=2][population>=0]::int_name
 /* 4.2 Town */
 
 node|z8[place=town][population>=40000],
-node|z9[place=town][population>=20000]
+node|z9[place=town][population>=20000],
+node|z10[place=town],
 {text: name;text-color: @city_label;}
 node|z8[place=town][population>=40000]::int_name,
-node|z9[place=town][population>=20000]::int_name
+node|z9[place=town][population>=20000]::int_name,
+node|z10[place=town]::int_name
 {text:int_name;text-color: @city_label;}
-node|z10-[place=town][population<20000],
+
```

Comment:
?

Reply:
what exactly?
I moved this rule from z10 to z11 to give room in z10 for towns >20k

Reply:
an empty line.

Reply:
Its just to separate sections to improve readability